README.md

```diff
@@ -1,2 +1,83 @@
-# wallet
-A budget handler application
+# Wallet Application 
+
+
+## Database deploy
+
+Database is provisioned using `docker-compose`. For that, following steps must be accomplished:
+
+1. Set wallet's database password by creating following environmental variable
+
+   ```bash
+   $ export MYSQL_PASSWD=<password>
+   ```
+
+   
+
+2. Set MySQL's root password by creating following environmental variable
+
+   ```bash
+   $ export MYSQL_ROOT_PASSWD=<password>
+   ```
+
+   
+
+3. Deploy (here we are in base dir)
```

Comment:
This step will need to change after you move the docker-compose file to the root of the repo.

Reply:
done